superset-frontend/src/explore/components/controls/DndColumnSelectControl/ColumnSelectPopoverTrigger.test.tsx

```diff
@@ -0,0 +1,185 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { configureStore } from '@reduxjs/toolkit';
+import { Provider } from 'react-redux';
+import {
+  fireEvent,
+  render,
+  screen,
+  userEvent,
+} from 'spec/helpers/testing-library';
+import { AdhocColumn } from '@superset-ui/core';
+import { ColumnMeta } from '@superset-ui/chart-controls';
+import ColumnSelectPopoverTriggerWrapper from './ColumnSelectPopoverTrigger';
+
+const createStore = (datasource = {}) =>
+  configureStore({
+    reducer: {
+      explore: () => ({
+        datasource,
+      }),
+    },
+  });
+
+const defaultColumns: ColumnMeta[] = [
+  {
+    column_name: 'test_column',
+    verbose_name: 'Test Column',
+    type: 'STRING',
+  },
+];
+
+const firstAdhocColumn: AdhocColumn = {
+  label: 'First Custom Column',
+  sqlExpression: '',
+  expressionType: 'SQL',
+};
+
+const secondAdhocColumn: AdhocColumn = {
+  label: 'Second Custom Column',
+  sqlExpression: 'SUM(amount)',
+  expressionType: 'SQL',
+};
+
+describe('ColumnSelectPopoverTriggerWrapper - Multiple Columns', () => {
```

Comment:
Could you avoid using `describe` and `it` and use `test` instead to follow our [testing guidelines](https://github.com/apache/superset/wiki/Testing-Guidelines-and-Best-Practices#avoid-nesting-when-youre-testing)?